plugins/reporters/fossid/src/main/kotlin/FossIdReporter.kt

```diff
@@ -96,7 +96,7 @@ class FossIdReporter : Reporter {
         return runBlocking(Dispatchers.IO) {
             val scanResults = input.ortResult.getScanResults().values.flatten()
             val scanCodes = scanResults.flatMapTo(mutableSetOf()) {
-                it.additionalData[SCAN_CODE_KEY].orEmpty().split(',')
+                it.additionalData[SCAN_CODE_KEY]?.split(',').orEmpty()
```

Comment:
Commit message nit: "*The* `FossIdReporter` ..."

Also in the commit message title the class name could be put in backticks.

Reply:
> Also in the commit message title the class name could be put in backticks.

The "Also" was referring to use backticks in the text, too, though 😉

Reply:
Backticks are cool. Here you go.